cmd/minishift/cmd/root.go

```diff
@@ -166,13 +166,22 @@ func setFlagsUsingViper() {
 	}
 }
 
-func init() {
+func processEnvVariables() {
 	enableExperimental, err := util.GetBoolEnv(enableExperimentalEnv)
 	if err == util.BooleanFormatError {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Error enabling experimental features: %s", err))
 	}
 	hasEnabledExperimental = enableExperimental
 
+	noProgressbar, err := util.GetBoolEnv(util.EnableNoProgressbarEnv)
+	if err == util.BooleanFormatError {
+		atexit.ExitWithMessage(1, fmt.Sprintf("Error enabling no progress bar action: %s", err))
+	}
+	util.HasEnabledNoProgressbar = noProgressbar
```

Comment:
`ShowProgressBars = !enableNoProgressBars`

Not sure if `util` is the correct place for this boolean. Sounds more like `config`